examples/CMakeLists.txt

```diff
@@ -243,6 +243,14 @@ if(WANT_CURL_EXAMPLE)
     endif(CURL_FOUND)
 endif(WANT_CURL_EXAMPLE)
 
+# The enet example won't compile if the enet library isn't available.
+# This example isn't important so it's disabled by default to prevent problems.
+option(WANT_ENET_EXAMPLE "Build ex_enet example" off)
```

Comment:
Yeah, we created a few of our own Find* packages. Probably shouldn't use that specific one, as it's probably licensed as GPL.

Just grab `cmake/FindDUMB.cmake` in the Allegro's source and modify it to search for enet instead.